app/services/users.js

```diff
@@ -13,3 +13,9 @@ exports.findBy = condition =>
     logger.error(`Failed to retrieve user from database. ${error}`);
     throw errors.databaseError(error.message);
   });
+
+exports.findAll = (limit = 5, offset = 0) =>
```

Comment:
Maybe `findAllBy` ? If tomorrow you need list users with specific lastName (for example), you should create another function for this.